### PR TITLE
Add line break functionality to instruction and description

### PR DESCRIPTION
**Pull request by iwootten**
### PR Context

Updates question description to be a list of strings in order to support multiple paragraphs in runner.

### Checklist

* [x] eq-translations updated to support any new schema keys which need translation - https://github.com/ONSdigital/eq-translations/pull/62

### CODE REVIEW

**Comment by iwootten** [comment]
[Add line break functionality in question description (M)](https://trello.com/c/On5OBCNm/3939-add-line-break-functionality-in-question-description-m)

File: schemas/common_definitions.json [hunk 1]
```diff
@@ -301,5 +301,11 @@
   "non_empty_string": {
     "type": "string",
     "minLength": 1
+  },
+  "string_list": {
```
Comment (ajmaddaford) [inline schemas/common_definitions.json:305]: As all the places this is used it's replacing `string_with_placeholders`, we should consider naming it `string_list_with_placeholders` and putting it in the string interpolation definitions file.

File: tests/schemas/valid/test_question_description.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    {
+      "id": "default-section",
+      "groups": [
+        {
+          "blocks": [
+            {
+              "type": "Question",
+              "id": "name-block",
+              "question": {
+                "description": ["Answer the question", "Go on"],
```
Comment (ajmaddaford) [inline tests/schemas/valid/test_question_description.json:39]: We should also add a placeholder in one of these strings.

File: schemas/common_definitions.json (hunk 1 above)
Comment (JamesGardiner) [reply schemas/common_definitions.json]: Or `string_array_with_placeholders` given it's JSON not Python?

**Pull request merged**